DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml

```diff
@@ -2,59 +2,73 @@
 <div class="content-container">
     <div class="filter-form">
         @{ Html.EnableClientValidation(false); }
-          <form action="@Model.Section.PreviousPageUrl" method="post" id="backForm">
-                 <div id="BackLink" class="breadcrumbs">
-                        <ol>
-                            <li>
-                                <button type="submit" class="button-link" id="filter-home">@(Model.Section.PageNumber == 1 ? "Home" : "Back")</button>
-                            </li>
-                        </ol>
-                </div>
-              @Html.HiddenFor(m => Model.Section.Name)
-              @Html.HiddenFor(m => Model.Section.SectionDataType)
-              @Html.HiddenFor(m => Model.OptionsSelected)
-              @Html.HiddenFor(m => Model.Section.PageNumber)
-              @Html.HiddenFor(m => Model.Section.SingleSelectOnly)
+        <form action="@Model.Section.PreviousPageUrl" method="post" id="backForm">
+            <div id="BackLink" class="govuk-breadcrumbs">
+                <a href="@Model.Section.PreviousPageUrl" class="govuk-back-link" id="filter-home">@(Model.Section.PageNumber == 1 ? "Home" : "Back")</a>
+                @*<button type="submit" class="govuk-back-link" id="filter-home">@(Model.Section.PageNumber == 1 ? "Home" : "Back")</button>*@
+            </div>
+            @Html.HiddenFor(m => Model.Section.Name)
+            @Html.HiddenFor(m => Model.Section.SectionDataType)
+            @Html.HiddenFor(m => Model.OptionsSelected)
+            @Html.HiddenFor(m => Model.Section.PageNumber)
+            @Html.HiddenFor(m => Model.Section.SingleSelectOnly)
         </form>
         <form action="@Model.Section.NextPageUrl" method="post" id="continueForm">
             <div class="active">
-                <h4>Step @Model.Section.PageNumber of @Model.Section.TotalNumberOfPages@if (@Model.Section.PageNumber > 1){<span> you have @Model.NumberOfMatches career matches</span>}</h4>
-                <h2>@Html.DisplayFor(model => Model.Section.Name)</h2>
-                @Html.Raw(Model.Section.Description)      
+                <span class="govuk-caption-xl">Step @Model.Section.PageNumber of @Model.Section.TotalNumberOfPages</span>
+                @*@if (@Model.Section.PageNumber > 1)
```

Comment:
Why is this removed? has this been moved to to correct HTML section?

Reply:
This was not needed as that part was not MVP and cant be done by 12th Nov. I removed that completely anyway instead of commenting it for now